src/main/kotlin/racingcar/domain/car/CarList.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain.car
+
+import racingcar.domain.record.RacingRecord
+
+data class CarList(
+    val cars: List<Car>
+) {
+    fun race(tryCount: Int) = RacingRecord(cars.map { car -> car.race(tryCount) })
```

Comment:
```suggestion
    fun race(tryCount: Int) {
        RacingRecord(cars.map { car -> car.race(tryCount) })
    }
```
expression body(=)를 잘 활용해주셨네요. 👍 
다만 리턴타입이 없는 경우 expression body를 사용하게 되면, 코드만 봤을 때에는 리턴타입이 존재하는 것으로 오해하기 쉽습니다.
리턴타입이 없는 경우에는 expression body를 지양해보면 어떨까요?

Reply:
CarList.race() 함수가 `RacingRecord`를 반환하는게 어색해서, 리턴타입이 없도록 변경했습니다!
`RacingGame.start()`가 RacingRecord를 반환하도록 변경했습니다.

함수 내용이 1줄이면 무분별하게 expression body를 사용했었는데, 이런 단점은 고려해보지 못했네요! 많이 배워갑니다 😀